### PR TITLE
Feature/download/cica 237

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**
Draft PR 
- added download functionality

### CODE REVIEW

File: page/download.summary.njk [hunk 1]
```diff
@@ -0,0 +1,554 @@
+{%- set assetUrl = assetUrl | default(assetPath) -%}
```
Comment (armoj) [inline page/download.summary.njk:1]: is this a "magic" variable used elsewhere? can't see it being used in this file

**Review by armoj — CHANGES_REQUESTED** [review]

File: questionnaire/routes.js [hunk 2]
```diff
@@ -20,6 +22,28 @@ router.route('/').get(async (req, res, next) => {
     }
 });
 
+router.route('/download-summary').get(async (req, res, next) => {
+    try {
+        const response = await qService.getSection(
+            req.cicaSession.questionnaireId,
+            'p--check-your-answers'
```
[the remark this answers is not in the document]
Comment (neil-stephen-mcgonigle) [reply questionnaire/routes.js:29]: 1. does this need to be hardcoded? **Yes** the downloaded application is generated from the check your answers page.

File: page/download.summary.njk (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply page/download.summary.njk:1]: Not sure what this is, I will address this when it comes to the next story which addresses the actual HTML.

File: package.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
         "test:dom": "jest --config=jest.config.jsdom.js",
         "test:node": "jest --config=jest.config.js",
         "coverage": "jest --coverage",
-        "build:dev": "npm run copy:css && npm run copy:js && npm run babel && npm run webpack:dev",
+        "build:dev": "npm run copy:css && npm run copy:js && npm run babel",
```
Comment (BarryPiccinni) [inline package.json:17]: I think this change was unintended.

File: package.json [hunk 4]
```diff
@@ -42,7 +42,7 @@
         "morgan": "~1.9.0",
         "nanoid": "^3.2.0",
         "nunjucks": "^3.2.1",
-        "q-transformer": "github:CriminalInjuriesCompensationAuthority/q-transformer#v4.2.0"
+        "q-transformer": "github:CriminalInjuriesCompensationAuthority/q-transformer"
```
Comment (BarryPiccinni) [inline package.json:45]: Specify a version/branch to avoid unexpected issues.

**Review by BarryPiccinni — CHANGES_REQUESTED** [review]
The changes to the UISchema appear to be missing?

**Review by BarryPiccinni — APPROVED** [review]

**Pull request merged**